pykern/pkcli/github.py

```diff
@@ -422,12 +415,17 @@ def restore(git_txz):
 
 
 class _Backup(GitHub):
-    def __init__(self):
+    def __init__(self, org):
+        def _repos():
+            if cfg.test_mode:
+                self.login()
+                return [self._github.repository(r[0], r[1]) for r in _TEST_REPOS]
```

Comment:
if _TEST_REPOS was list of PKDict then could say `r.name1, r.name2`. Not sure if really matters

Reply:
That's a good point. I'll make an issue. The bug I fixed related in github_test was related to this.

Reply:
Fixed this in #421